src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -96,7 +96,8 @@ internal void GenSspiClientContext(SspiClientContextStatus sspiClientContextStat
                 inSecurityBufferArray = Array.Empty<SecurityBuffer>();
             }
 
-            int tokenSize = MaxTokenSize;
```

Comment:
I think this is the only reference to MaxTokenSize, so we can remove the declaration at the top.